tests/sentry/api/endpoints/test_user_details.py

```diff
@@ -1,79 +1,55 @@
-from django.core.urlresolvers import reverse
-
 from sentry.models import Organization, OrganizationStatus, User, UserOption
 from sentry.testutils import APITestCase
 
 
 class UserDetailsTest(APITestCase):
-    # TODO(dcramer): theres currently no way to look up other users
```

Comment:
does this no longer apply? if it does or you're not sure, better to keep it

Reply:
It's hard to maintain code that's hidden in comments. The original version still lives in git history so I usually try to delete them when I see them. You're right though, I shouldn't remove the TODO.